mne/evoked.py

```diff
@@ -402,10 +402,11 @@ def plot(self, picks=None, exclude='bads', unit=True, show=True, ylim=None,
             channel traces will not be shown.
         window_title : str | None
             The title to put at the top of the figure window.
-        spatial_colors : bool
-            Color code lines by mapping physical sensor coordinates into color
-            values. Spatially similar channels will have similar colors.
-            Bad channels will be dotted.
+        spatial_colors : None | 'head' | 'skirt' | 'notopo'
```

Comment:
If you use grey color, you don't know which channel is bad, and if you have multiple bad channels, you don't know which position corresponds to which time course. If you do it with symbols (dashed line, x on legend), everything is better.